### PR TITLE
chore: add benchmarks

### PR DESCRIPTION
**Pull request by Uzlopak**
Add some reasonable benchmarks

### CODE REVIEW

**Comment by Uzlopak** [comment]
Actually it makes more sense to compare the functions and not the targets.. so benchmark-verify and benchmark-sign and not benchmark-node and benchmark-web.

File: test/benchmark-verify.bench.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { bench, describe } from "vitest";
+import { verify as verifyNode } from "../src";
+import { verify as verifyWeb } from "../src/web";
+import { toNormalizedJsonString } from "./common";
```
Comment (wolfy1339) [inline test/benchmark-verify.bench.ts:4]:
```suggestion
import { verify as verifyNode } from "../src/index.ts";
import { verify as verifyWeb } from "../src/web.ts";
import { toNormalizedJsonString } from "./common.ts";
```

File: test/benchmark-sign.bench.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { bench, describe } from "vitest";
+import { sign as signNode } from "../src";
+import { sign as signWeb } from "../src/web";
+import { toNormalizedJsonString } from "./common";
```
Comment (wolfy1339) [inline test/benchmark-sign.bench.ts:4]:
```suggestion
import { verify as verifyNode } from "../src/index.ts";
import { verify as verifyWeb } from "../src/web.ts";
import { toNormalizedJsonString } from "./common.ts";
```

File: test/common.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export function toNormalizedJsonString(payload: object) {
+  // GitHub sends its JSON with an indentation of 2 spaces and a line break at the end
+  const payloadString = JSON.stringify(payload, null, 2) + "\n";
```
Comment (wolfy1339) [inline test/common.ts:3]: This information isn't correct. It never was :man_shrugging: 
```suggestion
  // GitHub sends its JSON with no indentation and no line breaks at the end
  const payloadString = JSON.stringify(payload);
```

Comment (wolfy1339) [reply test/common.ts:3]: Would require new hashes for all the tests. Maybe not for now

**Review by wolfy1339 — APPROVED** [review]

**Pull request merged**